doc/apiary/v2/fiware-ngsiv2-cookbook.apib

```diff
@@ -49,29 +84,62 @@ This resource allows you to obtain a list of subresources published by the API:
             "temperature": 35.6
          },
          {
+            "type": "Car",
+            "id": "abc345fg",
+            "speed": 14.7
+         }
+        ]
+
+### Pagination [GET /v2/entities{?limit,offset,options}]
+
++ Parameters
+    + options: count
+    + limit: 2
+    + offset: 2
+
++ Response 200 (application/json)
+
+    + Headers
+
+            X-Total-Count: 30
+            Link: http://context.example.org/v2/entities?offset=4&limit=2; rel="next", http://context.example.org/v2/entities?offset=28&limit=2; rel="last", http://context.example.org/v2/entities?offset=0&limit=2; rel="first", http://context.example.org/v2/entities?offset=2&limit=2; rel="prev"
```

Comment:
Not sura how Apiary/FABRE will render this, but at least in the Github diff view it is quite long...